docs/quickstart.rst

```diff
@@ -105,22 +111,35 @@ These quick functions give you slightly less control over the machine learning w
 Walkthrough
 -----------
 
-Consider a regression analysis as a simple example of the use of visualizers in the machine learning workflow. Using a `bike sharing dataset <https://s3.amazonaws.com/ddl-data-lake/yellowbrick/bikeshare.zip>`_ based upon the one uploaded to the `UCI Machine Learning Repository <https://archive.ics.uci.edu/ml/datasets/bike+sharing+dataset>`_, we would like to predict the number of bikes rented in a given hour based on features like the season, weather, or if it's a holiday.
+Let's consider a regression analysis as a simple example of the use of visualizers in the machine learning workflow. Using a bike sharing dataset based upon the one uploaded to the `UCI Machine Learning Repository <https://archive.ics.uci.edu/ml/datasets/bike+sharing+dataset>`_, we would like to predict the number of bikes rented in a given hour based on features like the season, weather, or if it's a holiday.
 
-.. note:: We have updated the dataset from the UCI ML repository to make it a bit easier to load into Pandas; make sure you download the `Yellowbrick version of the dataset <https://s3.amazonaws.com/ddl-data-lake/yellowbrick/bikeshare.zip>`_.
+.. NOTE:: We have updated the dataset from the UCI ML repository to make it a bit easier to load into Pandas; make sure you download the Yellowbrick version of the dataset using the ``load_bikeshare`` method below. Please also note that Pandas is required to follow the supplied code. Pandas can be installed using ``pip install pandas`` if you haven't already installed it.
 
-After downloading the dataset and unzipping it in your current working directory, we can load our data as follows:
+We can load our data using the ``yellowbrick.datasets`` module as follows:
 
 .. code-block:: python
 
     import pandas as pd
+    from yellowbrick.datasets import load_bikeshare
+
+    X, y = load_bikeshare()
+    print(X.head())
```

Comment:
Using the new datasets module ensures that any changes to the datasets are reflected in the docs. Unfortunately, we do lose the nice CSV loading in Pandas which makes the workflow more realistic. However, I think it is more important that the quickstart code works correctly.